bad.php

```diff
@@ -0,0 +1,10 @@
+<?php
+class Thing_In_A_Box {
+    function do_thing($snake_var) {
```

Comment:
Method name "Thing_In_A_Box::do_thing" is not in camel caps format
Visibility must be declared on method "do_thing"
Opening brace should be on a new line